sklearnex/dispatcher.py

```diff
@@ -66,14 +67,22 @@ def get_patch_map():
         # kNN
         mapping.pop('knn_classifier')
         mapping.pop('kneighborsclassifier')
+        mapping.pop('knn_regressor')
+        mapping.pop('kneighborsregressor')
         mapping.pop('nearest_neighbors')
         mapping.pop('nearestneighbors')
         mapping['knn_classifier'] = [[(neighbors_module,
                                        'KNeighborsClassifier',
                                        KNeighborsClassifier_sklearnex), None]]
+        mapping['knn_regressor'] = [[(neighbors_module,
+                                      'KNeighborsRegressor',
+                                      KNeighborsRegressor_sklearnex), None]]
         mapping['nearest_neighbors'] = [[(neighbors_module,
                                           'NearestNeighbors',
                                           NearestNeighbors_sklearnex), None]]
+        mapping['kneighborsclassifier'] = mapping['knn_classifier']
+        mapping['kneighborsregressor'] = mapping['knn_regressor']
+        mapping['nearestneighbors'] = mapping['nearest_neighbors']

```

Comment:
Do these lines fix the unpatching issue?

Reply:
yes